src/azure-cli/azure/cli/command_modules/storage/operations/account.py

```diff
@@ -145,7 +149,9 @@ def update_storage_account(cmd, instance, sku=None, tags=None, custom_domain=Non
             directory_service_options='AADDS' if enable_files_aadds else 'None')
     if assign_identity:
         params.identity = Identity()
-
+    if enable_large_file_share:
+        LargeFileSharesState = cmd.get_models('LargeFileSharesState')
+        params.large_file_shares_state = LargeFileSharesState("Enabled")
```

Comment:
I find `create` and `update` have same code pieces.

Reply:
Right. For both create and update command, we just need to add a new property in the same way.

Reply:
Maybe we can abstract a function.
It's a general problem. `create` and `update` has lots of similar code. If we write them twice, it's error-prone and hard to maintain.

Reply:
But it is just a condition. How can we abstract a function for this? And for other properties, we also implement like this.